modules/cluster-wide-proxy-preqs.adoc

```diff
@@ -2,30 +2,35 @@
 //
 // * networking/configuring-cluster-wide-proxy.adoc
 
-:_content-type: PROCEDURE
+:_content-type: CONCEPT
 [id="cluster-wide-proxy-prereqs_{context}"]
 = Prerequisites for configuring a cluster-wide proxy
 
-To configure a cluster-wide proxy, you must meet the following requirements. These requirements are valid for both fresh installation and post installation proxy configuration.
+To configure a cluster-wide proxy, you must meet the following requirements. These requirements are valid when you configure a proxy during installation or post-installation.
 
+[discrete]
 [id="cluster-wide-proxy-general-prereqs_{context}"]
 == General requirements
 
 * You are the cluster owner.
 * Your account has sufficient privileges.
-* You have added the `ec2.<region>.amazonaws.com`, `elasticloadbalancing.<region>.amazonaws.com`, and `s3.<region>.amazonaws.com` endpoints to your virtual private cloud (VPC) endpoint. These endpoints are required to complete requests from the nodes to the AWS EC2 API. Because the proxy works on the container level, not the node level, you must route these requests to the AWS EC2 API through the AWS private network. Adding the public IP address of the EC2 API to your allowlist in your proxy server is not sufficient.
 ifdef::openshift-rosa[]
-* You have the `rosa` CLI installed and configured.
-endif::[]
+* You have an existing Virtual Private Cloud (VPC) for your cluster.
+endif::openshift-rosa[]
 ifdef::openshift-dedicated[]
-* You must have a Customer Cloud Subscription (CCS) cluster with a VPC that the proxy can access.
-* You have the `ocm` CLI installed and configured.
-endif::[]
+* You have an existing Virtual Private Cloud (VPC) for your cluster.
+* You are using the Customer Cloud Subscription (CCS) model for your cluster.
+endif::openshift-dedicated[]
+* The proxy can access the VPC for the cluster and the private subnets of the VPC. The proxy is also accessible from the VPC for the cluster and from the private subnets of the VPC.
+* You have added the `ec2.<region>.amazonaws.com`, `elasticloadbalancing.<region>.amazonaws.com`, and `s3.<region>.amazonaws.com` endpoints to your VPC endpoint. These endpoints are required to complete requests from the nodes to the AWS EC2 API. Because the proxy works at the container level and not at the node level, you must route these requests to the AWS EC2 API through the AWS private network. Adding the public IP address of the EC2 API to your allowlist in your proxy server is not sufficient.
 
+[discrete]
 [id="cluster-wide-proxy-network-prereqs_{context}"]
 == Network requirements
 
 * If your proxy re-encyrpts egress traffic, you must create exclusions to the domain and port combinations. The following table offers guidance into these exceptions.
++
+--
 ** Allowlist the following OpenShift URLs for re-encryption.
```

Comment:
Thank you for highlighting that. I have updated both of the steps in this module that include this format so that they state:

> Add ... to your allowlist